src/libNode/Node.h

```diff
@@ -160,7 +160,7 @@ class Node : public Executable {
       m_pendingTxnBuffer;
 
   std::mutex m_mutexTxnPacketBuffer;
-  std::vector<bytes> m_txnPacketBuffer;
+  std::map<bytes, bytes> m_txnPacketBuffer;
```

Comment:
Can we use unordered_map instead

Reply:
Don't think its of any gain considering we are going to store not more than 1 or 2 pkts.
Its O(logN) vs O(1). Considering N = 2, benefits over each other are negligible.
Also, if we go with unordered_map or unordered_set, we would need to have hash function for `bytes` which i think is over kill to do in such situation.